WcaOnRails/app/views/forum_topics/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<!-- NOTE: This is meant for displaying old content of the phpBB forum. It is DEPRECATED! -->
+
+<div class="container">
+  <h2><%= @topic.topic_title %></h2>
+  <% @topic.forum_posts.sort_by { |post| -post.post_time }.each do |post| %>
```

Comment:
I personally think it makes more sense to sort posts from oldest to newest, rather than newest to oldest as you've done here.